docs/section5/Rule5-49.md

```diff
@@ -0,0 +1,53 @@
+
+      - Check if surface is regulated, add zone total area of building envelope to building total: `if scc_dict_p[surface_p.id] != "UNREGULATED": building_total_envelope_area += sum(surface.area for surface in zone_p.surfaces)`
+
+    - Check if zone is conditioned or semi-heated, add zone air leakage rate to building total: `if zone_conditioning_category_dict_p[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]: building_total_air_leakage_rate += zone_p.infiltration.infiltration_flow_rate`
+
+      - Check if measured air leakage rate is not entered for zone, raise empty measured air leakage rate flag: `if NOT zone_p.infiltration.measured_air_leakage_rate: empty_measured_air_leakage_rate_flow_flag = TRUE`
+
+      - Else, measured air leakage rate is entered for zone, add measured air leakage rate to building total: `else: building_total_measured_air_leakage_rate += zone_p.infiltration.measured_air_leakage_rate`
+
+- Calculate the required proposed design air leakage rate at 75Pa: `target_air_leakage_rate_75pa_p = 0.6 * building_total_envelope_area`
+
+**Rule Assertion:**  
+
+- Case 1: For P_RMR, if the building total air leakage rate for conditioned and semi-heated zones is equal to the required proposed design air leakage rate at 75Pa with a conversion factor of 0.112 as per Section G3.1.1.4: `if building_total_air_leakage_rate == target_air_leakage_rate_75pa_p * 0.112: PASS`
+
+- Case 2: else if 1). the building total air leakage rate for conditioned and semi-heated zones is not equal to the required proposed design air leakage rate at 75Pa with a conversion factor of 0.112 as per Section G3.1.1.4, and 2). measured air leakage rate is not entered for all conditioned and semi-heated zones: `else if ( building_total_air_leakage_rate != target_air_leakage_rate_75pa_p * 0.112 ) AND ( empty_measured_air_leakage_rate_flow_flag ): UNDETERMINED and raise_message "THE BUILDING TOTAL AIR LEAKAGE RATE IS NOT EQUAL TO THE REQUIRED PROPOSED DESIGN AIR LEAKAGE RATE AT 75PA WITH A CONVERSION FACTOR OF 0.112 AS PER SECTION G3.1.1.4. AND MEASURED AIR LEAKAGE RATE IS NOT ENTERED FOR ALL CONDITIONED AND SEMI-HEATED ZONES. VERIFY THE PROPOSED AIR LEAKAGE RATE IS MODELED CORRECTLY."`
+
+- Case 3: else if 1). the building total air leakage rate for conditioned and semi-heated zones is not equal to the required proposed design air leakage rate at 75Pa with a conversion factor of 0.112 as per Section G3.1.1.4, and 2). the measured air leakage rate is entered for all conditioned and semi-heated zones, and 3). the building total air leakage rate is equal to the measured air leakage rate with a conversion factor of 0.112 as per Section G3.1.1.4: `else if ( building_total_air_leakage_rate != target_air_leakage_rate_75pa_p * 0.112 ) AND ( empty_measured_air_leakage_rate_flow_flag == FALSE ) AND ( building_total_air_leakage_rate == building_total_measured_air_leakage_rate * 0.112 ): PASS`
+
```

Comment:
Suggest also confirming that ReulsetModelInstance.measured_infiltration_pressure_difference=75

I also don't think we need RulesetModelInstance.is_measured_infiltration_based_on_test. If others agree, I'll post it as RMD issue.

Reply:
yes, I dont think we need that boolean. Not sure if that was added for another ruleset